spring-pulsar/pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.0.0-SNAPSHOT</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>io.streamnative.example</groupId>
+	<artifactId>spring-pulsar-example</artifactId>
+	<version>1.0.0</version>
+	<name>Spring Pulsar Example</name>
+	<description>Demo project for Spring integration with Apache Pulsar</description>
+	<properties>
+		<java.version>17</java.version>
+		<jfairy.version>0.6.5</jfairy.version>
+		<spring-pulsar.version>0.1.0-SNAPSHOT</spring-pulsar.version>
```

Comment:
If we end up pointing at M1 of course this will change to `0.1.0-M1`.

Reply:
Keeping this open as a reminder